### PR TITLE
Use py_zipkin and drop py26/py33 support

### PR DESCRIPTION
**Pull request by kaisen**
Removes the dependency on pyramid_zipkin and uses the new py_zipkin library instead. Also dropped py26 and py33 support to make it consistent with py_zipkin and pyramid_zipkin.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -5,20 +5,15 @@ swagger_zipkin |version|
...
+It is aimed to be a standalone package, with only dependency being
+`py_zipkin <https://github.com/Yelp/py_zipkin>`_.
 
 Quick Start
 -----------
 
     The decorator can be applied to any swagger-py or bravado client. The example
     shows a sample usage on bravado. It assume the code is being called from within
-    a pyramid view.
+a `py_zipkin.zipkin.zipkin_span` context manager or decorator.
```
Comment (bplotnick) [inline docs/source/index.rst:13]: I think this needs to be indented to be within the same block. But really I think you probably want the whole block to be dedented

File: docs/source/conf.py [hunk 2]
```diff
@@ -1,176 +1,41 @@
...
-#html_file_suffix = None
-
-# Output file base name for HTML help builder.
 htmlhelp_basename = 'swagger_zipkindoc'
 
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+
+intersphinx_mapping = {
+    'http://docs.python.org/': None
```
Comment (bplotnick) [inline docs/source/conf.py]: nit: comma

**Review by bplotnick — CHANGES_REQUESTED** [review]
Looks good! Just nit picking.

Also, can you update the PR description to include that you added py35?

**Comment by mjbryant** [comment]
Besides Ben's comments, looks good to me. 🚢

**Review by bplotnick — APPROVED** [review]
Looks good! :shipit:

**Pull request merged**